backend/src/models/db/tables.js

```diff
@@ -333,50 +340,52 @@ function initReviewTable(db) {
         }
     ]
 
-    const minRange = -1 // Between [minRange, minRange+maxRange]
+    const minRange = 4 // Between [minRange, maxRange]
     const maxRange = 20
     const numReviewTypes = reviewTypes.length
 
-    let reviews = []
+    // Prepare query
+    const columns = ['code', 'userID', 'recommend', 'enjoy', 'title', 'body']
+    const placeholders = columns.map(_ => '?').join()
+    const query = `INSERT INTO review (${columns}) VALUES (${placeholders})`
+    const prep = db.prepare(query)
+
+    let promises = []
 
     // For each of the courses
     for (const i in courseData) {
         // Get it's course code
         const code = courseData[i].code
         // Determine how many questions to add
-        const numReviews = Math.floor(Math.random() * maxRange + minRange)
+        const numReviews = getRandomIntInclusive(minRange, maxRange)
         if (numReviews <= 0) continue // not strictly necessary... but...meh
 
         // Now create each of the questions
         for (let i = 0; i < numReviews; i++) {
             // Determine the question type
-            const index = Math.floor(Math.random() * numReviewTypes)
+            const index = getRandomInt(0, numReviewTypes)
+            const uid = getRandomIntInclusive(1, NUM_DUMMY_USERS);
             // Create the question
             const review = {
                 code: code,
-                userID: Math.ceil(Math.random() * NUM_DUMMY_USERS),
-                recommend: Math.round(Math.random()),
-                enjoy: Math.round(Math.random() * (MAX_ENJOY - MIN_ENJOY) + MIN_ENJOY),
-                ...reviewTypes[index] }
+                userID: uid,
+                recommend: getRandomIntInclusive(0,1),
+                enjoy: getRandomIntInclusive(MIN_ENJOY, MAX_ENJOY),
```

Comment:
I feel like its odd that we don't have any fake data for the other categories like difficulty and workload. Maybe we should add that in?

Reply:
i forgot about this! will add now